src/gt4py/backend/gtc_backend/cuda/backend.py

```diff
@@ -49,19 +49,19 @@ def __init__(self, class_name, module_name, backend):
         self.module_name = module_name
         self.backend = backend
 
-    def __call__(self, ir: gtir.Stencil) -> Dict[str, Dict[str, str]]:
-        ir = GtirPipeline(ir).full()
-        base_oir = GTIRToOIR().visit(ir)
+    def __call__(self, stencil_ir: gtir.Stencil) -> Dict[str, Dict[str, str]]:
+        stencil_ir = GtirPipeline(stencil_ir).full()
+        base_o_ir = GTIRToOIR().visit(stencil_ir)
         oir_pipeline = self.backend.builder.options.backend_opts.get(
             "oir_pipeline", DefaultPipeline(skip=[NoFieldAccessPruning])
         )
-        oir = oir_pipeline.run(base_oir)
-        oir = FillFlushToLocalKCaches().visit(oir)
-        cuir = OIRToCUIR().visit(oir)
-        cuir = kernel_fusion.FuseKernels().visit(cuir)
-        cuir = extent_analysis.CacheExtents().visit(cuir)
+        o_ir = oir_pipeline.run(base_o_ir)
+        o_ir = FillFlushToLocalKCaches().visit(o_ir)
+        cu_ir = OIRToCUIR().visit(o_ir)
+        cu_ir = kernel_fusion.FuseKernels().visit(cu_ir)
+        cu_ir = extent_analysis.CacheExtents().visit(cu_ir)
```

Comment:
I don't think this is a good solution to the name clash problem.

Reply:
Fixed. Thanks!